<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QMainWindow" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0, 255, 127,0.3);
	border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>üîç</string>
    </property>
    <property name="icon">
     <iconset theme="address-book-new"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_searchCity">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background:rgba(170, 170, 255,0.5);
	border-radius:20px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cityName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Wearher in Nanchang</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:20px;
}</string>
    </property>
    <property name="text">
     <string>16.92‚ÑÉ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-image: url(:/icons/image2.jpg);
}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border:1px none white;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_date_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>4-9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êò®Â§©</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>4.43</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>ÊòéÂ§©</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>4..2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>4-42</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomWeather_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>Êô¥</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_date_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>4.3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_tomTem_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">Qlabel{
	font-size:15px;
	font-weight:bold;
}</string>
     </property>
     <property name="text">
      <string>2-18‚ÑÉ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>label_background</zorder>
   <zorder>pushButton_search</zorder>
   <zorder>lineEdit_searchCity</zorder>
   <zorder>label_cityName</zorder>
   <zorder>label_temperature</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
